.github/workflows/product-tests-specific-environment.yml

```diff
@@ -89,7 +113,31 @@ jobs:
         run: presto-product-tests/bin/run_on_docker.sh singlenode-postgresql -g postgresql_connector
       - name: Product Tests Specific 2.5
         run: presto-product-tests/bin/run_on_docker.sh singlenode-cassandra -g cassandra
+
+  product-tests-specific-environment2-2:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Cache local Maven repository
+        id: cache-maven
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-2-
+      - name: Populate maven cache
+        if: steps.cache-maven.outputs.cache-hit != 'true'
+        run: ./mvnw de.qaware.maven:go-offline-maven-plugin:resolve-dependencies
+      - name: Maven install
+        run: |
+          export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
+          ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl '!presto-docs,!presto-server-rpm,!presto-spark-package,!presto-spark-launcher,!presto-spark-testing'
       - name: Product Tests Specific 2.6
         run: presto-product-tests/bin/run_on_docker.sh singlenode-kerberos-hdfs-impersonation-with-wire-encryption -g storage_formats,cli,hdfs_impersonation,authorization
       - name: Product Tests Specific 2.7
-        run: presto-product-tests/bin/run_on_docker.sh singlenode-kafka -g kafka
+        run: presto-product-tests/bin/run_on_docker.sh singlenode-kafka -g kafka
```

Comment:
New line missing at the end of the file.

Reply:
Well, I actually have the line locally. Idk what's going on.

Reply:
Double-checked and added a new line at the end of the file.